nbs/09_API/02_maker.ipynb

```diff
@@ -495,7 +495,7 @@
     "        f.write(f\"# AUTOGENERATED! DO NOT EDIT! File to edit: {self.dest2nb}.\")\n",
     "        if last_future > 0: write_cells(cells[:last_future], self.hdr, f)\n",
     "        if self.parse: f.write(f\"\\n\\n# %% auto 0\\n__all__ = {all_str}\")\n",
-    "        write_cells(cells[last_future:], self.hdr, f, 1 if last_future>0 else 0)\n",
```

Comment:
This line of code where the `offset` was being set was causing some cell indexes to be wrong, and causing the wrong cell in the notebook to be overwritten upon `nbdev_update`.  This was causing some notebooks to be updated incorrectly when they had a `from __future__ ...`

When I removed this offset, everything now seems to be updating correctly.  However, I don't quite understand why this offset was here in the first place, so perhaps @jph00 this is something you might be able to help me understand?